ErrorBlocks.md

```diff
@@ -0,0 +1,156 @@
+</p></details>
+
+#### Creation of Void/Vacuum Volume and Graveyard
+
+_A method (implicit compliment) is implemented automatically during a DAG-MCNP5 run that creates a void volume in all spaces without geometry. This means that while one could explicitly create void volumes, it is often not necessary._
+<br/></br>
+_The vacuum and graveyard volumes are not part of the model being analyzed, but play a role in the final analysis. When particles move freely through the vacuum volume and stop once they reach the graveyard volume._
+
+| Steps  | In Graphic User Interface  | In Command Line  |
+|---|---|---|
+| __Create a cube large enough to envelop all other volumes.__  | - Select __Geometry -> Volume -> Create__ and select __Brick__ from the drop down menu. <br/></br> - Type 500 in each dimension box __(width, height, depth)__ and press __Apply__ to create the block.  | - The general form is `create [shape] [dimensions based on shape]`. </br> - For a simple cube 500x500x500 you can enter `create brick x 500`.  |
+| __Create another slightly larger cube to be used for graveyard.__  | - In command panel create another brick in the same way as part 1, using 750 as the new dimension in each dimension box.  | - Enter `create brick x 750`  |
+| __Center volumes within the larger cubes.__  | To move the blocks in command panel select __Geometry -> Volume -> Transform__ and select __Move__ from the drop down menu. <br/></br> - Type "5 6" into the __Volume ID(s)__ box. Then __Select Method -> Distance__ and type 100 into the __X Distance__ box. _This distance/direction may vary._  | - General form is `move vol [volume ids] [axis] [distance along axis]` <br/></br> - Enter `move vol 5 6 x 100`.  |
+| __Hollow out large cubes to match functionality.__  | - Select __Geometry -> Volume -> Boolean__ and select __Subtract__ from the dropdown menu. <br/></br> - Select volume 6 as volume A and volume 5 as volume B. Check __Keep Originals__ and click __Apply__. <br/></br> - You can then delete volume 6 by selecting __Geometry -> Volume -> Delete__ and typing 6 into the __Volume ID(s)__ box and clicking apply. <br/></br> - Select __Geometry -> Volume -> Boolean__ and select __Subtract__ from the dropdown menu. Select volume 5 as volume A and volumes 1,2,3, and 4 as volume B. Again make sure to check __Keep Originals__ and click __Apply__. <br/></br> - Then use the delete action to delete volume 5.  | - Enter `subtract vol 5 from 6 keep` <br/></br> - Enter `delete vol 6` to get rid of the original large box. <br/></br> - Enter `subtract vol 1 2 3 4 from 5 keep` or `subtract vol 1 to 4 from 5 keep`. <br/></br> - Enter `delete vol 5`. |
+
+<details><p><summary>Notes</summary>
+- These volumes could also be created in another CAD program and then assigned a material in the exact same way, but this is an example of how to create these volumes in Trelis and Cubit.

```

Comment:
You say "these volumes" but really you are only referring to the graveyard volume, correct? Because the implicit complement is created automatically upon exporting and should never have to be defined explicitly.